build/BundledRuntimes.props

```diff
@@ -1,23 +1,20 @@
 <Project>
   <PropertyGroup>
-    <ArchivePortableSuffix Condition=" '$(UsePortableSharedFramework)' == 'true' ">-portable</ArchivePortableSuffix>
-    <InstallerPortableSuffix Condition=" '$(UsePortableSharedFramework)' == 'true' AND '$(OSName)' != 'ubuntu' ">-portable</InstallerPortableSuffix>
-
     <!-- Downloaded Installers + Archives -->
-    <DownloadedSharedHostInstallerFileName Condition=" '$(InstallerExtension)' != '' ">dotnet-host-$(ProductMonikerRid).$(SharedHostVersion)$(InstallerPortableSuffix)$(InstallerExtension)</DownloadedSharedHostInstallerFileName>
+    <DownloadedSharedHostInstallerFileName Condition=" '$(InstallerExtension)' != '' ">dotnet-host-$(ProductMonikerRid)-$(SharedHostVersion)$(InstallerExtension)</DownloadedSharedHostInstallerFileName>
```

Comment:
I've just noticed that we have changed format of the asset name to use a `-` instead of a `.` separator between the RID and version.

The CLI still uses `.`. We need to be consistent across the SDK and the runtime assets on usage of `.` and `-` separators and format of the assets.  This is going to break the `dotnet-install` script as well.  See [https://github.com/dotnet/cli/blob/master/scripts/obtain/dotnet-install.ps1#L232](https://github.com/dotnet/cli/blob/master/scripts/obtain/dotnet-install.ps1#L232)

@chcosta - why was this change made to the assets?  I think we need to revert it in core-setup.  There will be too many things that will break, and I don't feel this is worth the breaking change.

/cc @livarcocc